jenkins/databricks/run_it.sh

```diff
@@ -20,71 +20,20 @@
 #   TEST_TAGS=xxx
```

Comment:
one problem might be that csputil does not download common script, so we may need to change csputil defaults to fetch also common-vars script along w/ the run_it script